src/DurableTask.Core/Query/OrchestrationQueryResult.cs

```diff
@@ -0,0 +1,35 @@
+﻿//  ----------------------------------------------------------------------------------
+//  Copyright Microsoft Corporation
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//  ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace DurableTask.Core.Query
+{
+    /// <summary>
+    /// The status of all orchestration instances with paging for a given query.
+    /// </summary>
+    public class OrchestrationQueryResult
+    {
+        /// <summary>
+        /// Gets or sets a collection of statuses of orchestration instances matching the query description.
+        /// </summary>
+        /// <value>A collection of orchestration instance status values.</value>
+        public IEnumerable<OrchestrationState> OrchestrationState { get; set; }
```

Comment:
To follow best practices, let's change the signature to use a more useful type and to be immutable:
```suggestion
        public IReadOnlyCollection<OrchestrationState> OrchestrationState { get; }
```

This change will require you to add a constructor so that you can assign this property when the object is created. You'll also want to remove "or sets" from the `<summary>` XML comment text since the property will be read-only from the user's perspective.